src/wp-admin/contribute.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Contribute administration panel.
+ *
+ * @package WordPress
+ * @subpackage Administration
+ */
+
+/** WordPress Administration Bootstrap */
+require_once __DIR__ . '/admin.php';
+
+// Used in the HTML title tag.
+$title = __( 'Get Involved' );
+
+list( $display_version ) = explode( '-', get_bloginfo( 'version' ) );
+
+require_once ABSPATH . 'wp-admin/admin-header.php';
+?>
+<div class="wrap about__container">
+
+	<div class="about__header">
+		<div class="about__header-title">
+			<h1>
+				<?php _e( 'Get Involved' ); ?>
+			</h1>
+		</div>
+
+		<div class="about__header-text">
+			<?php _e( 'Be the future of WordPress' ); ?>
+		</div>
+	</div>
+
+	<nav class="about__header-navigation nav-tab-wrapper wp-clearfix" aria-label="<?php esc_attr_e( 'Secondary menu' ); ?>">
+		<a href="about.php" class="nav-tab"><?php _e( 'What&#8217;s New' ); ?></a>
+		<a href="credits.php" class="nav-tab"><?php _e( 'Credits' ); ?></a>
+		<a href="freedoms.php" class="nav-tab"><?php _e( 'Freedoms' ); ?></a>
+		<a href="privacy.php" class="nav-tab"><?php _e( 'Privacy' ); ?></a>
+		<a href="contribute.php" class="nav-tab nav-tab-active" aria-current="page"><?php _e( 'Get Involved' ); ?></a>
+	</nav>
+
+	<div class="about__section has-2-columns is-wider-right">
+		<div class="column about__image">
+			<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%231d35b4' /%3E%3C/svg%3E" alt="" />
+		</div>
+		<div class="column is-vertically-aligned-center">
+			<p>
+				<?php
+				printf(
+					/* translators: %s: Percentage value. Note: "%%" is used to display an escaped percentage sign. */
+					__( 'Do you use WordPress for work, for personal projects, or maybe even just for fun? You can help shape the long-term success of the open source project that powers %s%% of the web.' ),
+					43
+				);
+				?>
+			</p>
+			<p>
+				<?php
+				_e( 'Join the WordPress contributor community and connect with others who are passionate about maintaining a free and open web.' );
+				?>
+			</p>
```

Comment:
```suggestion
			<p><?php _e( 'Join the WordPress contributor community and connect with others who are passionate about maintaining a free and open web.' ); ?></p>
```

Make it single line.